xmpp/Dockerfile

```diff
@@ -1,3 +1,3 @@
 FROM ghcr.io/processone/ejabberd:22.05
 
-COPY --chown=ejabberd ./ejabberd.yml /opt/ejabberd/conf/ejabberd.yml
+COPY --chown=ejabberd ./ejabberd.yml /home/ejabberd/conf/ejabberd.yml
```

Comment:
WARNING! I think this is a "revert" from a commit that should not be here!
The "/opt" is the right path.
Same thing in the `xmpp/ejabberd.yml` file